render/Camera.cpp

```diff
@@ -212,7 +212,9 @@ int Camera::AABBInsideFrustum(Vector3& vMin, Vector3& vMax, s32 worldX, s32 worl
 	for (int i=0; i<np; i++)
 	{
 		Vector3 signFlip(SIGN(p[i].a), SIGN(p[i].b), SIGN(p[i].c));
-		if ( p[i].Dot(vCen + vExt*signFlip) < -p[i].d )
+        Vector3 arg = vCen + vExt*signFlip;
+
+		if ( p[i].Dot(arg) < -p[i].d )
```

Comment:
Can we come up with a better name than arg? I know it isn't like the code is much better. ;)

vArg is better, but if you know of anything better, please do.

Reply:
Sure, I just wanted to make it compile, then get to renaming variables. I have no idea what these one letter guys mean, but will look at that :)

Reply:
Point taken, we'll look at it later. ;)